Sources/Danger/GitLabDSL.swift

```diff
@@ -135,7 +135,7 @@ extension GitLab {
             case downvotes
             case firstDeployedToProductionAt = "first_deployed_to_production_at"
             case forceRemoveSourceBranch = "force_remove_source_branch"
-            case id // swiftlint:disable:this identifier_name
+            case id 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitLabDSL.swift;
  Line: 138;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)